assets/data/projects.json

```
@@ -0,0 +1,33 @@
+[
+    {
+        "img": "/assets/img/logo.png",
+        "name": "AntAlmanac",
+        "desc": "AntAlmanac is a service made by students for students to create and organize your schedule with ease, and has far better functionality than traditional alternatives like WebSOC or Antplanner.",
```

Simplified the AntAlmanac description. 
```suggestion
        "desc": "AntAlmanac is a schedule planning and course exploration tool for UCI students.",
```

---

Committed your suggestion!